### ISSUE TITLE
Critical dependency AJV

### ISSUE THREAD

**Issue by ketysek** [eclipsesource/jsonforms#929]
Hi, I've installed jsonforms in my react application and now webpack is throwing those errors:

```
./node_modules/ajv/lib/compile/index.js
13:21-34 Critical dependency: the request of a dependency is an expression
./node_modules/ajv/lib/async.js
96:20-33 Critical dependency: the request of a dependency is an expression
webpackHotDevClient.js:136 ./node_modules/ajv/lib/async.js
119:15-28 Critical dependency: the request of a dependency is an expression
```

You're using ajv version "ajv": "^4.11.2", so it installs 4.11.8, which is a bit old. Don't you think that update to the newer version will resolve this annoying error?

**Comment by edgarmueller** [comment]
Yes, we'll do soon, thanks for bringing it up.

**Comment by edgarmueller** [comment]
Fixed via #947, will be released with 2.0.1
